primitives/bignumber.go

```diff
@@ -369,11 +390,14 @@ func (u *U128) LTE(v *U128) (r int) {
 	return
 }
 
-func (u U128) toJSON() string {
+func (u U128) ToJSON() string {
 	u1 := big.Int(u)
 	innerArrOfU := u1.Bytes()
 	var test = make([]byte, 2*len(innerArrOfU))
 	hex.Encode(test, innerArrOfU[:])
+	if string(test) == "" {
```

Comment:
When the `test` became `""`?
Since the content of the `test` is hex-encoding of integer, shouldn't it have some value always?

Reply:
There is. This change is some kind of bug fix.
`UXXX` object wraps `big.Int` and `(big.Int).bytes()` returns empty slice when the value of `big.Int` is `0`
Here's an example https://play.golang.org/p/wD1wTUej7cI

Reply:
Aha. Then how about using `len(test) == 0` condition instead of `string(test) == ""`?

Reply:
https://stackoverflow.com/questions/18594330/what-is-the-best-way-to-test-for-an-empty-string-in-go

 Both ways are fine. But I think `len(test)==0` is better. I didn't thought  that 😅